Gopkg.toml

```diff
@@ -21,3 +21,7 @@
 [[constraint]]
   name = "github.com/kubevirt/device-plugin-manager"
   branch = "master"
+
+[[constraint]]
+  name = "gopkg.in/goose.v2"
```

Comment:
I would vote for another lib with more developers.
Maybe https://github.com/gophercloud/gophercloud ?